lib/conjur/debify.rb

```diff
@@ -5,6 +5,16 @@
 
 include GLI::App
 
+config_file '.debifyrc'
+
+desc 'Set an environment variable (e.g. TERM=xterm) when starting a container'
+flag [:env], :multiple => true
+
+desc 'Mount local bundle to reuse gems from previous installation'
+default_value true
+switch [:'local-bundle']
```

Comment:
As of 1.11.1-beta12, Docker For Mac has some performance issues when sharing files between the Mac and a container. This switch turns off reusing gems from a previous bundle run, which speeds up `debify test` considerably.